app/controllers/api/users.php

```diff
@@ -647,6 +733,35 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
         ]), Response::MODEL_LOG_LIST);
     });
 
+App::get('/v1/users/:userId/targets')
```

Comment:
Let's add query support here

Reply:
as discussed we cannot add query support here because it fetches target from user document